### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kimleeng**
Hey I think my codes great, please check the logic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # MicrobeSeq-Denmark
+
+being reviewed
```
Comment (kimleeng) [inline README.md]: Be more descriptive in this

Comment (kimleeng) [reply README.md]: Ya I agree with this guy

Comment (kimleeng) [reply README.md]: I think this is issue #1

Comment (kimleeng) [reply README.md]: maybe @thej-ssi is keen on this

**Review by kimleeng — COMMENTED** [review]
Overall mediocore

**Comment by thej-ssi** [comment]
This should be a one-liner

**Pull request merged**